cmd/s2i/s2i.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/openshift/source-to-image/pkg/cmd/cli"
+)
+
+func main() {
+	rand.Seed(time.Now().UTC().UnixNano())
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
```

Comment:
looks like it was cargo-culted from origin:
https://github.com/openshift/origin/blob/master/cmd/openshift/openshift.go#L30

i think it's a harmless (if probably unnecessary) optimization